source/PhysicalQuantity/Force.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace PhpUnitsOfMeasure\PhysicalQuantity;
+
+use PhpUnitsOfMeasure\AbstractPhysicalQuantity;
+use PhpUnitsOfMeasure\UnitOfMeasure;
+use PhpUnitsOfMeasure\HasSIUnitsTrait;
+
+class Force extends AbstractPhysicalQuantity
+{
+	use HasSIUnitsTrait;
+
+	protected static $unitDefinitions;
+
+	protected static function initialize()
+	{
+
+		// newton
+		$newton = UnitOfMeasure::nativeUnitFactory('N');
+		$newton->addAlias('newton');
+		static::addUnit($newton);
+
+		static::addMissingSIPrefixedUnits(
+			$newton,
+			1,
+			'%pN',
+			[
+				'%Pnewton',
+				'%Pnewtons'
+			]
+		);
+
+		// kilonewton
+		$newUnit = UnitOfMeasure::linearUnitFactory('kN', 1000);
+		$newUnit->addAlias('kilonewton');
+		$newUnit->addAlias('kilonewtons');
+		static::addUnit($newUnit);
+
+		// kip
+		$newUnit = UnitOfMeasure::linearUnitFactory('kips', 4.4482216);
+		$newUnit->addAlias('kip');
```

Comment:
let's reverse the names here, and have `kip` be the unit name, and `kips` be the alias, to be consistent with the rest of the units we have.